src/lib/components/BucketAggregation/BucketAggregationValues.js

```diff
@@ -152,7 +154,6 @@ const ValueElement = ({
       <List.Item key={bucket.key}>
```

Comment:
Given that the id was the same as the `key`, then the key value is also wrong in that case as there is duplicate value.

Reply:
This one does not seem to cause an issue. Note that they are on different branches.